test/Microsoft.Framework.Internal.Test/BufferedHtmlContentTest.cs

```diff
@@ -131,20 +150,22 @@ public void WriteTo_WritesAllItems()
 
             // Assert
             Assert.Equal(2, content.Entries.Count);
-            Assert.Equal("Written from TestHtmlContent: HelloTest", writer.ToString());
+            Assert.Equal("Written from TestHtmlContent: HelloHtmlEncode[[Test]]", writer.ToString());
         }
 
+        // We're purposely avoiding anything here that actually gets encoded. We don't want to take a dependency
+        // on the actual encoder implementation.
```

Comment:
Why not use `CommonTestEncoder`? This test project already references the `Microsoft.Framework.WebEncoders.Testing` package.